ZUGFeRD/InvoiceDescriptor.cs

```diff
@@ -295,6 +295,23 @@ public class InvoiceDescriptor
         public DateTime? BillingPeriodEnd { get; set; }
 
 
+        /// <summary>
+        /// Spezification Identifier
+        /// </summary>
+        public string SpecificationId { get; set; }
+
+        /// <summary>
+        /// Details about the associated order confirmation
+        /// </summary>
+        public SellerOrderReferencedDocument SellerOrderReferencedDocument { get; set; } = new SellerOrderReferencedDocument();
+
+        /// <summary>
+        /// Exchange rate
```

Comment:
Can you add more documentation how to use this property?

Reply:
done